packages/frontend/src/components/common/ResourceView/ResourceViewGrid/index.tsx

```diff
@@ -47,9 +48,9 @@ const ResourceViewGrid: FC<ResourceViewGridProps> = ({
             .filter((group) => group.items.length > 0);
     }, [groups, items]);
 
-    const [draggableItems, setDraggableItems] = useState(
-        groupedItems.map((g) => g.items),
-    );
+    // const [draggableItems, setDraggableItems] = useState(
+    //     groupedItems.map((g) => g.items),
+    // );
```

Comment:
this can be removed then? A bit confused about some changes on this PR...

Reply:
@joaoviana pinning drag and drop items is work in progress. variable was unused. commented out to not remove code and fix the linting error